File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.java
class Solution {
    public int removeDuplicates(int[] nums) {

        int i = 1;
        int k = 0;

        for (int index = 0; index < nums.length; index++ ){
            if(nums[index] != nums[i - 1]){
                nums[i++] = nums[index];                 
            }
            k ++;
        }
        return i;


        
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-1.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        memory = counter = 0
        #res = a cnt y cnt es igual a 0
        
        for num in nums:
            if num == 0:
                memory = max(memory, counter)
                counter = 0
            else:
                counter += 1
        return max(memory, counter)